src/main/scala/net/ripe/rpki/publicationserver/AppConfig.scala

```diff
@@ -59,6 +60,17 @@ class AppConfig {
 
   def deltaUrl(sessionId: String, serial: Long) =
     rrdpRepositoryUri + "/" + sessionId + "/" + serial + "/delta.xml"
+
+  lazy val writeRsync = isMode("rsync")
```

Comment:
I would prefer two boolean flags because in my experience those are easier to get right when setting them (case insensitive, config file implicitly lists all possible values, etc).

Reply:
Will try it this way.